sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/lro/AnalyzeExtractiveSummarization.java

```diff
@@ -0,0 +1,105 @@
+                                         .buildClient();
+
+        List<TextDocumentInput> documents = new ArrayList<>();
+        for (int i = 0; i < 21; i++) {
+            documents.add(new TextDocumentInput(Integer.toString(i),
+                "The government of British Prime Minster Theresa May has been plunged into turmoil with "
+                    + "the resignation of two senior Cabinet ministers in a deep split over her Brexit strategy."
+                    + "The Foreign Secretary Boris Johnson, quit on Monday, hours after the resignation late on"
+                    + "Sunday night of the minister in charge of Brexit negotiations, David Davis.  Their"
+                    + "decision to leave the government came three days after May appeared to have agreed a"
+                    + "deal with her fractured Cabinet on the UK's post-Brexit relationship with the EU."
+                    + "That plan is now in tatters and her political future appears uncertain."
+                    + "May appeared in Parliament on Monday afternoon to defend her plan, minutes after"
+                    + "Downing Street confirmed the departure of Johnson. May acknowledged the splits in"
+                    + "her statement to MPs, saying of the ministers who quit: \"We do not agree about the"
+                    + "best way of delivering our shared commitment to honoring the result of the referendum.\""
+                    + "The Prime Minister's latest plitical drama began late on Sunday night when Davis quit,"
+                    + "declaring he could not support May's Brexit plan.  He said it involved too close a "
+                    + "relationship with the EU and gave only an illusion of control being returned to the UK"
+                    + "after it left the EU. \"It seems to me we're giving too much away, too easily, and"
+                    + "that's a dangerous strategy at this time,\" Davis said in a BBC radio interview Monday"
+                    + "morning. Johnson's resignation came Monday afternoon local time, just before the Prime"
+                    + " Minister was due to make a scheduled statement in Parliament. \"This afternoon, the Prime"
+                    + "Minister accepted the resignation of Boris Johnson as Foreign Secretary,\" a"
+                    + "statement from Downing Street said."
+            ));
+        }
+
+        SyncPoller<AnalyzeActionsOperationDetail, AnalyzeActionsResultPagedIterable> syncPoller =
+            client.beginAnalyzeActions(documents,
+                new TextAnalyticsActions().setDisplayName("{tasks_display_name}")
+                    .setExtractSummaryActions(
+                        new ExtractSummaryAction()
+                            .setMaxSentenceCount(2)
+                            .setSentencesOrderBy(SummarySentencesOrder.RANK)),
+                new AnalyzeActionsOptions().setIncludeStatistics(false),
```

Comment:
isn't the default value `false`? why do u pass it then?

Reply:
Yes. The default value is `false`, i can remove it.